pkg/capabilities/consensus/ocr3/datafeeds/data_feeds_types.proto

```diff
@@ -9,6 +9,8 @@ message DataFeedsOutcomeMetadata {
 }
 
 message DataFeedsMercuryReportInfo {
-  uint32 timestamp = 1;
-  double price = 2;
+  // DEPRECATED = 1;
+  // DEPRECATED = 2;
+  int64 observation_timestamp = 3;
+  int64 benchmark_price = 4;
```

Comment:
nit: @bolekk since we're not actually persisting these fields anywhere (and we're not migrating server + client separately) you can safely overwrite the field.